modules/vaos/app/serializers/vaos/appointment_requests_serializer.rb

```diff
@@ -44,6 +44,7 @@ class AppointmentRequestsSerializer
                :booked_appt_date_time,
                :type_of_care_id,
                :friendly_location_name,
+               :cc_appointment_request,
```

Comment:
is there anything clever we can do here to still have the json evaluate strictly? It might not be possible to do, but I'm asking because VAMF has a tendency to just drop null-valued keys. Wondering if doing a reverse_merge or something to ensure it's always at least an empty hash would allow you to still evaluate with `strict: true`

Reply:
got it to evaluate strictly using `oneOf` in the schema. I think this is better for the FE otherwise they'll be checking for empty hashes rather than null 😄

Reply:
sounds good! as long as we're not just dropping the root level key, I'm ok with that. Root level objects should ideally always be null or object, not just dropped. 😂